doc/manpages/bob-build-dev.rst

```diff
@@ -74,6 +74,22 @@ Options
     packages=<packages regex>
       download modules that match a given regular expression, build all other.
 
+``--download-layer MODE``
+    Download from binary archive for layer (yes, no, forced)
+
+    Acts like ``--download`` but only for the modules of the layer that match a
+    given regular expression (``--download`` option will be overwritten for
+    matching modules).
+    Can be used multiple times (if regex is used also multiple times the last mode wins).
+    A sub layer is separated with a dot.
```

Comment:
I would rather separate layers with `/`. The dot is a valid character for a package, so this may lead to ambiguous matches. It's also a regex meta character that would need escaping to match precisely.

Reply:
Changed.